pkg/kubectl/polymorphichelpers/attachablepodforobject.go

```diff
@@ -25,7 +25,7 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
-	"k8s.io/kubernetes/pkg/controller"
+	"k8s.io/kubernetes/pkg/kubectl/util/podutils"
```

Comment:
@seans3 would you mind creating a followup and rename that to just `pod` to align that with what I wrote in https://github.com/kubernetes/kubernetes/pull/69692#discussion_r226307268 ?